### PR TITLE
Add Treafik HTTP/3 Support

### PR DESCRIPTION
**Pull request by wuast94**
Added documentation on how to enable HTTP/3 Support when using Traefik reverse proxy.

I found #3515 , if im not wrong we need to make http3 support configurable when using reverse proxy?

Also i can test against http3 support and it works but i think it only test for general support not if a service is using it activley ? I`m new to this topic

### CODE REVIEW

File: reverse-proxy.md [hunk 1]
```diff
@@ -556,61 +556,64 @@ The examples below define the dynamic configuration in YAML files. If you rather
...
-   
-            nextcloud-chain:
-                chain:
-                    middlewares:
-                        # - ... (e.g. rate limiting middleware)
-                        - https-redirect
-                        - nextcloud-secure-headers
+      routers:
+        nextcloud:
+          rule: "Host(`<your-nextcloud-domain>`)"
```
Comment (tar-xz) [inline reverse-proxy.md:588]: Can you replace it with `<your-nc-domain>`?

**Comment by wuast94** [comment]
Found another one and changed it too 😊

**Comment by szaimen** [comment]
Hi @wuast94 thanks for the PR and sorry for taking so long to review it! Can you please rebase your branch and resolve the conflicts? Otherwise we are not abke to merge this. Thanks a lot!

**Comment by wuast94** [comment]
well, im not a github pro 😅 hope i fixed all. can you check @szaimen ?

**Comment by szaimen** [comment]
Thanks! @wuast94 is there a reason why you switched from 4 spaces to 2 spaces for the indentation of the yaml files?

**Comment by wuast94** [comment]
i always work with 2 whitespaces, its compacter and makes files more readable, also i dont like to throw a away my precious screenspace. and in this case, the official Traefik docs are using 2 spaces too.

can change it back to 4 if wanted 😊 2 or 4 spaces is a never ending diskussion i think 😅 @szaimen

File: reverse-proxy.md [hunk 2]
```diff
@@ -556,61 +556,68 @@ The examples below define the dynamic configuration in YAML files. If you rather
...
+          redirectscheme:
+            scheme: https 
+
+        nextcloud-chain:
+          chain:
+            middlewares:
+              # - ... (e.g. rate limiting middleware)
+              - https-redirect
+              - nextcloud-secure-headers
+
```
Comment (szaimen) [inline reverse-proxy.md:620]:
```suggestion
```

**Review by szaimen — COMMENTED** [review]
LGTM but one feedb

File: reverse-proxy.md [hunk 3]
```diff
@@ -556,61 +556,67 @@ The examples below define the dynamic configuration in YAML files. If you rather
     # STATIC CONFIGURATION
    
     entryPoints:
-        https:
-            address: ":443" # Create an entrypoint called "https" that uses port 443
+      https:
+        address: ":443" # Create an entrypoint called "https" that uses port 443
+        # If you want to enable HTTP/3 support, see below
+        http3: {}
```
Comment (szaimen) [inline reverse-proxy.md:562]:
```suggestion
        # If you want to enable HTTP/3 support, uncomment the line below
        # http3: {}
```

File: reverse-proxy.md [hunk 4]
```diff
@@ -556,61 +556,67 @@ The examples below define the dynamic configuration in YAML files. If you rather
...
-        file:
-            directory: "/path/to/dynamic/conf" # Adjust the path according your needs.
-            watch: true
+      file:
+        directory: "/path/to/dynamic/conf" # Adjust the path according your needs.
+        watch: true
+
+    # Enable HTTP/3 feature, don`t forget to route 443 UDP to Traefik (Firewall\NAT\Traefik Container)`
+    experimental:
+      http3: true
```
Comment (szaimen) [inline reverse-proxy.md:579]:
```suggestion
    # Enable HTTP/3 feature, don`t forget to route 443 UDP to Traefik (Firewall\NAT\Traefik Container) by uncommenting the lines below`
    # experimental:
      # http3: true
```

**Review by szaimen — COMMENTED** [review]
Thanks! Two ideas

**Comment by wuast94** [comment]
done @szaimen

File: reverse-proxy.md [hunk 5]
```diff
@@ -556,61 +556,67 @@ The examples below define the dynamic configuration in YAML files. If you rather
...
    
     providers:
-        file:
-            directory: "/path/to/dynamic/conf" # Adjust the path according your needs.
-            watch: true
+      file:
+        directory: "/path/to/dynamic/conf" # Adjust the path according your needs.
+        watch: true
+
+    # Enable HTTP/3 feature, don`t forget to route 443 UDP to Traefik (Firewall\NAT\Traefik Container) by uncommenting the lines below`
```
Comment (szaimen) [inline reverse-proxy.md:577]:
```suggestion
    # Enable HTTP/3 feature by uncommenting the lines below. Don't forget to route 443 UDP to Traefik (Firewall\NAT\Traefik Container)
```

**Review by szaimen — APPROVED** [review]
Thanks, LGTM but didnt test

**Comment by wuast94** [comment]
done

**Pull request merged**